neighborhood_data/fetch_images.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+# encoding=utf8
+
+"""
+Fetch image metadata and download thumbnails.
+"""
+
+import csv
+import errno
+import os
+import requests
+import urllib
+
+DESCRIPTIONS_CSV = 'neighborhood_descriptions.csv'
+OUTPUT_FILE = 'neighborhood_extended_descriptions.csv'
+STATUS_FILE = 'image_status.csv'
+
+STATUS_FIELDS = ['zipcode', 'source_field', 'url', 'issue']
+
+if not os.path.isfile(DESCRIPTIONS_CSV):
+    print('\nMissing neighborhood images and descriptions in {f}.\n\n'.format(
+        f=DESCRIPTIONS_CSV))
+    raise IOError(errno.ENOENT,
+                  os.strerror(errno.ENOENT),
+                  DESCRIPTIONS_CSV)
+
+IMAGE_DIRECTORY = 'images/'
```

Comment:
I had to manually create this directory before the script would run, it might be helpful to have the script check for it and create it if necessary, or create it in the `update_data.sh` scrpt.

Reply:
I added a `.gitkeep` file to put the directory in the repository, but must have not gotten that in before gitignoring the directory contents. I'll add the directory explicitly.